internal/infrastructure/kubernetes/configmap.go

```diff
@@ -0,0 +1,179 @@
+package kubernetes
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"text/template"
+
+	corev1 "k8s.io/api/core/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/envoyproxy/gateway/internal/ir"
+)
+
+var (
+	// envoyGatewayService is the name of the Envoy Gateway service.
+	envoyGatewayService = "envoy-gateway"
+)
+
+var envoyTmpl = template.Must(template.New(envoyCfgFileName).Parse(`
+admin:
+  access_log_path: /dev/null
+  address:
+    socket_address:
+      address: 127.0.0.1
```

Comment:
address and port value should be templated

Reply:
https://github.com/envoyproxy/gateway/issues/212 tracks this requirement.